package.json

```diff
@@ -43,8 +43,8 @@
     "@shutter-network/shutter-crypto": "0.1.0-beta.3",
     "@snapshot-labs/lock": "^0.1.1015",
     "@snapshot-labs/pineapple": "^0.1.0-beta.1",
-    "@snapshot-labs/snapshot.js": "^0.4.93",
-    "@snapshot-labs/tune": "^0.1.25",
+    "@snapshot-labs/snapshot.js": "0.4.94",
+    "@snapshot-labs/tune": "0.1.25",
```

Comment:
Still missing the `^`

Reply:
Looking at current issue I feel like better not to use `^` - hard to revert 🥲

Reply:
But yea maybe in a separate PR